src/main.rs

```diff
@@ -128,87 +126,72 @@ fn main() {
         Some(Commands::Init {}) => {
             get_or_create_gen_dir();
             println!("Gen repository initialized.");
+            Ok(())
         }
-        Some(Commands::Import(cmd)) => match r#gen::commands::import::execute(&cli_context, cmd) {
-            Ok(_) => {}
-            Err(err) => {
-                println!("Error importing: {err}");
-            }
-        },
+        Some(Commands::Import(cmd)) => Ok(r#gen::commands::import::execute(&cli_context, cmd)?),
         Some(Commands::Update(cmd)) => {
             r#gen::commands::update::execute(&cli_context, cmd);
+            Ok(())
         }
         Some(Commands::Export(cmd)) => {
             r#gen::commands::export::execute(&cli_context, cmd);
+            Ok(())
         }
-        Some(Commands::Remote(cmd)) => match handle_remote_command(&operation_conn, &cmd) {
-            Ok(_) => {}
-            Err(err) => {
-                eprintln!("Remote command failed: {err}");
-                std::process::exit(1);
-            }
-        },
-
+        Some(Commands::Remote(cmd)) => Ok(handle_remote_command(&operation_conn, &cmd)?),
```

Comment:
This seems a little weird to me.  If handle_remote_command returns an error, will the Ok() wrap the error?

Reply:
The Err will return out. What is happening:
handle_remote_command()? returns Result or Err. If err, bail, If result, unwrap it. The cli_call function returns Result<(), Err>, so the `Ok(...)` makes it compatible with that.